app/models/backend_api_config.rb

```diff
@@ -14,9 +14,7 @@ class BackendApiConfig < ApplicationRecord
     backend_api.metrics.each { |metric| metric.send(:sync_backend) }
   end
 
-  sifter :path_desc do
-    System::Database.oracle? ? 'path DESC NULLS LAST' : {path: :desc}
-  end
+  scope :with_subpath, -> { common_query = where.not(path: '/'); System::Database.oracle? ? common_query.where('path is NOT NULL') : common_query.where.not(path: '') }
```

Comment:
This is meant to be temporary so we can merge and unblock many PRs depending on this one.